backend/src/main/java/ai/verta/modeldb/entities/versioning/RepositoryEntity.java

```diff
@@ -214,10 +204,6 @@ public Integer getRepository_visibility() {
     return repository_visibility;
   }
 
-  public boolean isProtected() {
-    return Objects.equals(repositoryAccessModifier, RepositoryAccessModifier.PROTECTED.getNumber());
-  }
```

Comment:
we should have the flag just that we should not have the enum in protos

Reply:
Ok, Got it. then column related fixes we should keep.